src/lib/app-state-hoc.jsx

```diff
@@ -76,6 +77,9 @@ const AppStateHOC = function (WrappedComponent, localesOnly) {
                         } else {
                             initializedGui = initTutorialCard(initializedGui, tutorialId);
                         }
+                    } else if (props.showPreviewInfo) {
```

Comment:
This is a small thing, but would it be possible to rewrite these if/elses to be phrased positively rather than negatively? Like:

```
if (tutorialId === null) {
    if (props.showPreviewInfo) {...}
} else { // tutorial is present
...
}
```

I find it much easier to read code like that... otherwise I have to say to myself, "ok, this will run if tutorial is not null is false, so tutorial IS null, AND showPreviewInfo is true...